src/boot/bootctl.c

```diff
@@ -1313,6 +1349,7 @@ static int help(int argc, char *argv[], void *userdata) {
                "                       variables cannot be written\n"
                "     --make-machine-id-directory=yes|no|auto\n"
                "                       Create $BOOT/$MACHINE_ID\n"
+               "  -a --arch=ARCH       EFI arch for installed binaries\n"
```

Comment:
abbreviation seems unnecessary. I mean you have it a short option "a" already, no need to abbreviate the long option.